gateway-package/src/main/resources/bin/start.sh

```diff
@@ -160,6 +160,8 @@ fi
 ATTLS_ENABLED="false"
 if [ -n "$(echo ${ZWE_configs_spring_profiles_active:-} | awk '/^(.*,)?attls(,.*)?$/')" ]; then
     ATTLS_ENABLED="true"
+    ZWE_configs_server_ssl_enabled="false"
+    ZWE_configs_server_internal_ssl_enabled="false"
```

Comment:
maybe the internal should be separate? the use case is not 100% clear for me, but from the settings perspective we instruct to have a separate keystore configuration than the main one

Reply:
I guess as a default behaviour it makes sense, anyway i have added possibility to redefine the value by `ZWE_configs_server_internal_ssl_enabled="${ZWE_configs_server_internal_ssl_enabled:-false}"`